src/ert/storage/migration/block_fs.py

```diff
@@ -103,6 +104,10 @@ def migrate_case(storage: StorageAccessor, path: Path) -> None:
         name=path.name,
         ensemble_size=ensemble_size,
         iteration=iteration,
+        metadata={
+            "migration_ert_version": __version__,
+            "migration_from": "block_fs",
```

Comment:
Add storage migration version as well? Would be good to have it for 1 - > 2 also

Reply:
I thought about it, but I didn't want to introduce changing metadata functionality in this PR.